testonly/hammer/hammer.go

```diff
@@ -761,6 +758,23 @@ func (s *hammerState) getSMRRev(ctx context.Context, prng *rand.Rand) error {
 	if !reflect.DeepEqual(root, smrRoot) {
 		return fmt.Errorf("get-smr-rev(@%d)=%+v, want %+v", rev, root, smrRoot)
 	}
+
+	if err := s.validateSMRMatchesWrittenContents(root); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (s *hammerState) validateSMRMatchesWrittenContents(root *types.MapRootV1) error {
+	mapContents := s.prevContents.PickRevision(root.Revision)
+	wantRootHash, err := mapContents.RootHash(s.cfg.MapID, s.vc.Hasher)
```

Comment:
Same idea here, there is not need for such a long name in such a short function. Something like `want` would do.